src/main/java/com/questhelper/helpers/quests/romeoandjuliet/RomeoAndJuliet.java

```diff
@@ -120,7 +120,7 @@ public void setupSteps()
 		giveLetterToRomeo = new NpcStep(this, NpcID.ROMEO, new WorldPoint(3211, 3422, 0), "Bring the letter to Romeo in Varrock Square.", letter);
 		talkToLawrence = new NpcStep(this, NpcID.FATHER_LAWRENCE, new WorldPoint(3254, 3483, 0), "Talk to Father Lawrence in north east Varrock.");
 		talkToLawrence.addDialogStep("Ok, thanks.");
-		talkToApothecary = new NpcStep(this, NpcID.APOTHECARY, new WorldPoint(3195, 3405, 0), "Bring the Apothecary cadava berries in south west Varrock.", cadavaBerry);
+		talkToApothecary = new NpcStep(this, NpcID.APOTHECARY, new WorldPoint(3195, 3405, 0), "Bring the Apothecary cadava berries in south east Varrock.", cadavaBerry);
```

Comment:
Maybe this reads better?
```suggestion
		talkToApothecary = new NpcStep(this, NpcID.APOTHECARY, new WorldPoint(3195, 3405, 0), "Bring the cadava berries to the Apothecary in south east Varrock.", cadavaBerry);
```